fault/verilator_target.py

```diff
@@ -37,13 +37,14 @@
 
 class VerilatorTarget(Target):
     def __init__(self, circuit, actions, directory="build/",
-                 flags=[], skip_compile=False, include_verilog_files=[],
-                 magma_output="verilog"):
+                 flags=[], skip_compile=False, include_verilog_libraries=[],
+                 include_directories=[], magma_output="verilog"):
```

Comment:
What is the use case for this flag? Is there a simple test case for it?

A small note documenting the difference between these 2 flags would be great.

Reply:
It allows your specify a directory that contains verilog files you want to include.

`include_verilog_libraries` corresponds to the `-v` flag in verilator
`include_directories` corresponds to the `-I` flag in verilator

I think the functionality is pretty much the same, but it's convenient to provide both options depending on what the user has ready (a file name or a directory path). That's what I could glean from the verilator help string from https://www.veripool.org/projects/verilator/wiki/Manual-verilator

```
     -I<dir>                    Directory to search for includes
...
     -v <filename>              Verilog library
...
```

Reply:
One issue is that when using `-I`, verilator searches for verilog files with the same name as the module it's trying to include. When you pass a file `-v`, verilator loads all the modules in that file into the namespace, so the semantics are slightly different (e.g. you can't use -I if your module name doesn't match the file name it's contained in).

Reply:
I suppose you could achieve this using the `flags` parameter, so this is really a convenience method for not having to build the `-I` strings, and instead you can just pass a list of directories.

Reply:
Added a test

Reply:
Sounds good to me. Thanks for the test.